modules/programs/waybar.nix

```diff
@@ -370,15 +370,13 @@ in {
             "Highly customizable Wayland bar for Sway and Wlroots based compositors.";
           Documentation = "https://github.com/Alexays/Waybar/wiki";
           PartOf = [ "graphical-session.target" ];
+          After = [ "graphical-session.target" ];
         };
 
         Service = {
-          Type = "dbus";
-          BusName = "fr.arouillard.waybar";
           ExecStart = "${cfg.package}/bin/waybar";
-          Restart = "always";
-          RestartSec = "1sec";
-          KillMode = "mixed";
```

Comment:
I'd rather keep this logic.
Making this a dbus service allows systemd to monitor the for the health of the service by checking whether the associated bus name is registered.

Reply:
See #1852 for the `KillMode` logic.

Reply:
I think the dbus clauses were removed upstream because waybar doesn't actually expose a dbus interface, it's simply a dbus client. https://github.com/Alexays/Waybar/pull/801

Reply:
TIL. In that case we could only preserve the `KillMode`?

Reply:
Done!